apps/etterlatte-saksbehandling-ui/client/src/shared/statusbar/Statusbar.tsx

```diff
@@ -65,3 +69,20 @@ const Name = styled.div`
   margin-right: auto;
   margin-left: 0.5em;
 `
+
+const Skeleton = styled.div`
```

Comment:
Hvorfor ikke bruke designsystemene sine?

Reply:
For å bruke det må vi hente siste versjon av aksel (vi er fortsatt på ds-react) og det er vesentlig mer jobb.